docs/guides/pushsecrets.md

```diff
@@ -10,6 +10,9 @@ By default, the secret created in the secret provided will not be deleted even a
 {% include 'full-pushsecret.yaml' %}
 ```
 
+!!! note
```

Comment:
I would have expected to see this on the AWS PushSecret section since it affects only that provider.